src/components/05_pages/NodeForm/index.js

```diff
@@ -142,17 +149,23 @@ class NodeForm extends React.Component {
     return (
       <Fragment>
         <PageTitle>Create {this.props.bundle}</PageTitle>
-        {this.props.schema && (
-          <Paper>
-            <div className={styles.container}>
-              <FormControl margin="normal" fullWidth>
-                {Object.entries(this.props.uiMetadata)
-                  .map(([fieldName, { widget, inputProps }]) => {
-                    if (Widgets[widget]) {
+        {this.props.schema &&
+          this.props.uiSchema && (
+            <Paper>
+              <div className={styles.container}>
+                <FormControl margin="normal" fullWidth>
+                  {Object.entries(
+                    createUISchema(
+                      this.props.uiSchema.fieldSchema,
+                      this.props.uiSchema.formDisplaySchema,
+                      this.props.widgets,
+                    ),
+                  )
+                    .map(([fieldName, { widget, inputProps }]) => {
                       // @todo We need to pass along props.
                       // @todo How do we handle cardinality together with jsonapi
                       // making a distinction between single value fields and multi value fields.
-                      const fieldSchema = this.getSchemaInfo(
+                      const entitySchema = this.getSchemaInfo(
```

Comment:
I'm confused about the new name, it is certainly not the schema of the entire entity, but just of one field.

Reply:
sorry!